Packs/MicrosoftExchangeOnPremise/Integrations/EWSv2/EWSv2.py

```diff
@@ -645,13 +649,19 @@ def get_message_for_body_type(body, body_type, html_body):
     Returns:
         Body: the body of the message.
     """
+    demisto.debug(f"get_message_for_body_type: Received body_type={body_type}, handle_inline_image={handle_inline_image}")
```

Comment:
```suggestion
    demisto.debug(f"get_message_for_body_type: Received {body_type=}, {handle_inline_image=}")
```
Up to you whether to change, this is in order not to have future changes that will print a different argument name and value that will lead to misunderstanding.